specs/protocol/derivation.md

```diff
@@ -744,19 +744,21 @@ To interact with the engine, the [execution engine API][exec-engine] is used, wi
 - `engine_newPayload`
   - [`engine_newPayloadV2`] — executes a Bedrock/Canyon/Delta execution payload to create a block.
   - [`engine_newPayloadV3`] — executes an Ecotone execution payload to create a block.
+  - [`engine_newPayloadV4`] - executes an Isthmus execution payload to create a block.
 
-The current version of `op-node` uses the `v3` Engine API RPC methods as well as `engine_newPayloadV2`, due to
-`engine_newPayloadV3` only supporting Ecotone execution payloads. Both `engine_forkchoiceUpdatedV3` and
+The current version of `op-node` uses the `v4` Engine API RPC methods as well as `engine_newPayloadV3`, due to
```

Comment:
We'll still use V2 for pre-Ecotone blocks.

Reply:
@meyer9 do we want to keep the V2 comment around? The current version will still use V2.